plugin.rb

```diff
@@ -999,6 +1001,9 @@ def schema
     def run
       check_xhr unless params[:download]
       query = DataExplorer::Query.find(params[:id].to_i)
+      query.last_run_at = Time.now
+      query.save
```

Comment:
This should be `query.save!`so that we raise an error if any.

Reply:
Doing that returns `NoMethodError - undefined method save! for #<DataExplorer::Query:0x000056235ba9abf8>`. 
I believe this is a built-in function, any ideas?

Reply:
I see, we have implemented the `save` function in `DataExplorer::Query` manually. Ignore.

Just commit the `last_run_at` change we discussed above and feel free to merge the PR.

Reply:
Will do, thanks! :)